src/modules/News/routes.ts

```diff
@@ -24,4 +24,10 @@ router.get(
   newsController.getTopHeadlines,
 );
 
+router.get(
+  '/single',
```

Comment:
As the single articles is related to user saved data, we could add here the authMiddleware.

Reply:
Added in the news route /single